docs/charts/bar.md

```diff
@@ -107,11 +107,18 @@ This setting is used to avoid drawing the bar stroke at the base of the fill.
 In general, this does not need to be changed except when creating chart types
 that derive from a bar chart.
 
+**Note:** for negative bars in vertical chart, `top` and `bottom` are flipped. Same goes for `left` and `right` in horizontal chart.
+
 Options are:
 * `'bottom'`
 * `'left'`
 * `'top'`
 * `'right'`
+* `false`
+
+#### borderWidth
+
+If this value is a number, it is applied to all sides of the rectangle (left, top, right, bottom), except [`borderSkipped`](#borderskipped). If this value is an object, the `left` property defines the left border width. Similarly the `right`, `top` and `bottom` properties can also be specified. Omitted borders and [`borderSkipped`](#borderskipped) are skipped.
```

Comment:
I think it'd be more intuitive for omitted borders to use the default. You can set it to `0` if you'd like to skip the border

Reply:
Default is `0`, so would not change anything if not overwritten. Is that what you were thinking of?

So lets iterate:
```js
Chart.defaults.global.datasets.bar.borderWidth = 4;
var chart = new Chart(ctx, {
    type: 'line',
    data: {
        datasets: [{
            data: [0, 1, 2, 3],
            borderWidth: {left: 1, right: 1}
        }]
    },
    options: {
        elements: {
            rectangle: {
                borderWidth: {top: 10}
            }
        }
    }
}
```
Which one should this resolve to:
```js
borderWidth: {bottom: 4, left: 1, top: 4, right: 1}
borderWidth: {bottom: 4, left: 1, top: 10, right: 1}
borderWidth: {bottom: 0, left: 1, top: 0, right: 1}
```

Reply:
Currently this does not inherit from any upper level values, and it is consistent with how `padding` works. I think these should be consistent, so if inheriting from upper level values, then padding should do the same and that would be breaking change.

Reply:
Oh, I didn't realize the default was `0`